average.js

```diff
@@ -1,6 +1,15 @@
-
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+    let result = 0;
+    let countNan = 0;
+    
+    numbers.forEach((number) => isNaN(number) ? countNan+=1 : result+=number);
+    
+    // Sensor Malfunction check!
+    if(countNan > Math.ceil(0.30*numbers.length)){
```

Comment:
Nice and clean implementation :+1: 
Any reason or reference to choose 30% as the tolerance for the number of NaN readings?

Reply:
Normally when I do Modelling/data science projects with my team we have found that if missing value is in range of 25%-30%, those missing values can be imputed using statistical methods & it tends to give more robust results while analysis and modelling. Beyond this threshold, we found the result getting biased and we simply used to drop the variable for analysis.

I would also love to hear your preference while choosing the tolerance.